### ISSUE TITLE
Wrong value in calibrate.c for magnetometer

### ISSUE THREAD

**Issue by thomasnugent** [psiphi75/esp-mpu9250#5]
Hey Simon,

I think there's a small typo on line 307 of calibrate.c:

float avg_radius = (v_avg.x + v_avg.y + v_avg.z) / 2.0; // should be divided by 3

**Comment by psiphi75** [comment]
Yes, I think you are right.  Do you want to create a pull request for that?
